core/src/test/java/tech/tablesaw/TableAssertions.java

```diff
@@ -27,4 +30,48 @@ public static void assertTablesEquals(Table compareWith, Table sortedTable) {
       }
     }
   }
+
+  public static void assertTableEquals(
+      String[] expectedHeaders, Object[][] expectedRowValues, Table actual) {
+    assertColumnNamesEquals(expectedHeaders, actual);
+    assertRowValuesEquals(expectedRowValues, actual);
+  }
+
+  public static void assertColumnNamesEquals(String[] expectedColumnNames, Table actual) {
```

Comment:
It looks like these methods aren't used outside of this class, so you could make them private. Also, it'd be nice to check that the table names are equal I think